pkg/autoscaler/statserver/server_test.go

```diff
@@ -189,6 +190,45 @@ func TestServerDoesNotLeakGoroutines(t *testing.T) {
 	}
 }
 
+func TestServerWithOwnershipStatsReceived(t *testing.T) {
```

Comment:
Is this accurate? Shouldn't we also add a test that checks for a bucket-host **and** ownership=true?

Reply:
I think the name is confusing. Renamed and add one more test covering for bucket-host and ownership=true.